python/runtime/cudaq/algorithms/py_state.cpp

```diff
@@ -158,6 +158,70 @@ void bindPyState(py::module &mod) {
                 reinterpret_cast<std::complex<double> *>(info.ptr), info.size));
           },
           "Return a state from data.")
+      .def_static(
+          "from_data",
+          [](std::vector<py::buffer> tensors) {
```

Comment:
Is `py::buffer` shared_ptr like? Should this be pass by reference?

Reply:
absolutely. Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1711/commits/a1f4cb93b1991d52988975d8d8b2ea094fc335f0